### PR TITLE
Iss3 : Initial Weather Data analysis

### PR DESCRIPTION
**Pull request by samgdotson**
closes iss3

### CODE REVIEW

File: weather/tests/test_data_funcs.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+from data_funcs import isfloat
+from data_funcs import to_float
+
+
+# ===================================================================
+# Tests for isfloat()
+# ===================================================================
+
```
Comment (kamuda1) [inline weather/tests/test_data_funcs.py]: Are you going to use pytest? If so, you can use pytest.mark.parametrize to condense each test case into one function.

File: weather/tests/test_data_funcs.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    string = "42a"
+    a_float = isfloat(string)
+    assert not a_float
+
+
+# ===================================================================
+# Test for to_float()
+# ===================================================================
+
+dataframe = {'integers': [11, 250, 39, 42],
```
Comment (kamuda1) [inline weather/tests/test_data_funcs.py]: If using pytest, you can make this dataframe a pytest fixture and give it to each test function.

File: weather/tests/test_data_funcs.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+             'floats': ["0.91", "0.0001", 1.0, -777.2],
+             'mixed': ["x87", "0.1x", "T", "11"]}
+
+dataframe_pd = pd.DataFrame(dataframe)
+
+
+def test_to_float1():
```
Comment (kamuda1) [inline weather/tests/test_data_funcs.py]: These cases can be condensed into one function with a pytest.mark.parametrize decorator.

File: weather/tests/test_data_funcs.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+from data_funcs import isfloat
+from data_funcs import to_float
```
Comment (kamuda1) [inline weather/tests/test_data_funcs.py]: Once cairo is a package you can use the format `from package.module import function` to import these. Right now these don't load.

File: weather/weather_proc.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+from data_funcs import to_float
+
+
+if __name__ == '__main__':
+
+    # import raw data -- the raw data is available on the box website.
+
+    path = "/home/dotson/Documents/cairo/willard_weather_data.csv"
```
Comment (katyhuff) [inline weather/weather_proc.py]: Instead of this, consider using wget and the URL

File: weather/data_funcs.py [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    boolean
+    """
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def to_float(dataframe, column):
```
Comment (kamuda1) [inline weather/data_funcs.py]: Why don't you use to_nemeric, astype, or infer_objects? See [this](https://stackoverflow.com/questions/15891038/change-data-type-of-columns-in-pandas) discussion on changing pandas columns to floats. As someone who has 'reinvented the wheel' many times, I can tell you it's not worth it.

**Comment by samgdotson** [comment]
Closing pull request, deprecated

**Pull request closed without merging**